keras/preprocessing/image.py

```diff
@@ -358,6 +358,19 @@ def img_to_array(img, data_format=None):
     return x
 
 
+def save_img(path, x, data_format=None, scale=True):
```

Comment:
I think we should add the arguments "file_format" and "**kwargs" (passed to `Image.save()`) https://pillow.readthedocs.io/en/3.1.x/reference/Image.html#PIL.Image.Image.save

"format" needs to be renamed "file_format" to avoid confusion with "data_format".

Reply:
Added `file_format` and `**kwargs` parameters which are passed to the `Image.save` method.